governance/votes/Trial Executive vote - Aug 2, 2023.md

```diff
@@ -0,0 +1,142 @@
+
+- The DIRECT-SPARK-DAI [Maximum Debt Ceiling (line)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) will be increased by 180 million DAI from 20 million DAI to 200 million DAI
+
+#### Spark Proxy Spell
+
+As per this [poll](https://vote.makerdao.com/polling/QmZyFH21#poll-detail) the following changes to the Spark Protocol DAI Direct Deposit Module (D3M) will be made via Spark Protocol proxy spell defined at [0x443f3f4328553f5f85dFc0BA3D59969708201E14](https://etherscan.io/address/0x443f3f4328553f5f85dFc0BA3D59969708201E14#code).
+
+If this executive proposal passes, the spell will execute resulting in the following changes:
+
+- The new DAI market Interest Strategy will be set to track the ETH-C stability fee instead of the DSR rate.
+- The DAI market [Loan-To-Value (LTV)](https://docs.aave.com/risk/asset-risk/risk-parameters#loan-to-value) and [Liquidation Threshold](https://docs.aave.com/risk/asset-risk/risk-parameters#liquidation-threshold) will be set to 0.01% - This will effectively remove DAI as a collateral type.
+- The WETH market [Reserve Factor](https://docs.aave.com/risk/asset-risk/risk-parameters#reserve-factor) will be set to 5% - This will reduce the protocol fee taken on the ETH market giving suppliers a larger APY.
+- The WETH market [Variable Rate Slope](https://docs.aave.com/risk/liquidity-risk/borrow-interest-rate#variable-interest-rate-model-parameters) 1 will be set to 3% - This sets the optimal borrow rate at 4%, based on the `variableRateSlope1` change and `baseVariableRate` being at 1%.
+
+### Harbor Trade Credit Vault (RWA004-A) Debt Ceiling and tell Call
+
+As per this [poll](https://vote.makerdao.com/polling/QmR8cYb1#poll-detail) the following changes to the Harbor Trade Credit Vault (RWA004-A) Debt Ceiling parameters will be made, if this executive proposal passes:
+
+- The Debt Ceiling will be decreased by 7 million DAI from 7 million DAI to 0 DAI.
+
+In addition, `tell` will be called on the RWA Liquidation Oracle for this vault type; this places the vault into a “soft liquidation” and is a technical requirement for Centrifuge to commence the unwinding process.
+
+Please review the discussion [thread](https://forum.makerdao.com/t/request-to-poll-decrease-debt-ceiling-for-harbor-trade-credit-htc-drop-to-0/21373) to help inform your position before voting.
+
+### New Silver (RWA002-A) Debt Ceiling, Stability Fee
+
+As per this [poll](https://vote.makerdao.com/polling/QmaU1eaD) the following changes to the New Silver Vault (RWA002-A) Debt Ceiling parameters will be made, if this executive proposal passes:
+
+- The Debt Ceiling will be increased by 30 million DAI from 20 million DAI to 50 million DAI.
+- The Stability Fee will be increased by 3.5% from 3.5% to 7%.
+
+In addition, the DOC variable for this deal will be: QmTrrwZpnSZ41rbrpx267R7vfDFktseQe2W5NJ5xB7kkn1 and the document can be viewed [here](https://gateway.pinata.cloud/ipfs/QmTrrwZpnSZ41rbrpx267R7vfDFktseQe2W5NJ5xB7kkn1).
```

Comment:
Prefer human readable language for parameters. So 'RWA Agreement' rather than doc. and link to the relevant docs here: <https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement>